pythran/tables.py

```diff
@@ -873,6 +873,12 @@ def update_effects(self, node):
     logger.warn("Pythran support disabled for module: omp")
     del MODULES["omp"]
 
+# copyto and isclose are not always supported by numpy
```

Comment:
maybe add the version of numpy that cause the issue in the comment here, otherwise the whol thing looks like: why do we support this at all?